rasa/nlu/utils/__init__.py

```diff
@@ -56,13 +64,42 @@ def write_to_file(filename: Text, text: Any) -> None:
 
 
 def build_entity(
-    start: int, end: int, value: Text, entity_type: Text, **kwargs: Dict[Text, Any]
+    start: int,
+    end: int,
+    value: Text,
+    entity_type: Text,
+    role: Optional[Text] = None,
+    group: Optional[Text] = None,
+    **kwargs: Dict[Text, Any],
 ) -> Dict[Text, Any]:
     """Builds a standard entity dictionary.
 
-    Adds additional keyword parameters."""
+    Adds additional keyword parameters.
 
-    entity = {"start": start, "end": end, "value": value, "entity": entity_type}
+    Args:
+        start: start position of entity
+        end: end position of entity
+        value: text value of the entity
+        entity_type: name of the entity type
+        role: role of the entity
+        group: group of the entity
+        **kwargs: additional parameters
+
+    Returns: an entity dictionary
```

Comment:
minor, but it goes in the next line (we really need a tool to validate this)
```suggestion
    Returns:
        an entity dictionary
```